CHANGELOG.md

```diff
@@ -16,3 +16,7 @@ Ravelin sends the work `null` in responses if there are no changes.  This causes
 # 0.1.9
 
 Added support for tags.
+
+# 0.1.19
```

Comment:
This changelog is weird... 0.1.4 is in the middle then versions go up in both directions. Maybe worth reordering? (Normally newer things are at the top.)